src/CodeEditor/tests.jsx

```diff
@@ -53,7 +53,7 @@ describe( 'CodeEditor', () =>
             wrapper.setProps( { onMouseOver } );
 
             expect( wrapper.find( `.${cssMap.editor}` )
-                .prop( 'onMouseOver' ) ).to.equal( onMouseOver );
+                .prop( 'onMouseEnter' ) ).to.equal( onMouseOver );
```

Comment:
I think we should change the test descriptions for clarity:
- `should pass onMouseOver to the container div` -> `should pass onMouseOver to the container div as onMouseEnter`
- `should pass onMouseOut to the container div` - > `should pass onMouseOut to the container div as onMouseLeave`